stdlib/bigarray.mli

```diff
@@ -174,7 +174,9 @@ val int16_unsigned : (int, int16_unsigned_elt) kind
 (** See {!Bigarray.char}. *)
 
 val int : (int, int_elt) kind
-(** See {!Bigarray.char}. *)
+(** See {!Bigarray.char} and {!section:1_Elementkinds}.
+    The element size matches the OCaml [int] type, and not the C [int] type.
```

Comment:
I think you can simply repeat the description that this is the kind of OCaml integers array.

> Beware that this kind represents an array of OCaml [int], and not an array of 32 bits [int] a la C.

Reply:
I've copied the description from the element kinds, and simplified the reference to `C`: "they do not match the C [int] type".

The other element kinds have names that are very similar to types from C, which is probably where the confusion comes from. Emphasizing the distinction might be useful, given that it has already led to bugs in projects like `ctypes` (see original description).